src/styles/navigation_item.css

```diff
@@ -11,11 +13,33 @@
   white-space: nowrap;
   color: var(--grey-08);
   text-decoration: none;
-  text-transform: capitalize;
-  cursor: pointer;
   outline: none;
 }
 
+/* Force pointer on clickable elements*/
+[role=banner] .coz-nav-item a[role=menuitem],
+[role=banner] .coz-nav-item button[role=menuitem] {
+  cursor: pointer;
+}
+
+/* Hover effect */
+[role=banner] .coz-nav-item a[role=menuitem]:hover:before,
```

Comment:
Not sure, it could work without it for now I guess, but the original selector for this kind of element is `[role=banner] .coz-nav-item [role=menuitem]` and I guess @m4dz added the class selector to give some higher specificity. 
Because the attribute selector is very low on specificity, those elements could inherit some style that this declaration wouldn't overwrite.

So basically, today it seems to be ok to remove it but it's not future proof. Don't forget that there isn't CSS modules on the Cozy-bar and therefore no scoping at all.
So I wouldn't remove it.